lib/writeGltf.js

```diff
@@ -24,7 +23,7 @@ function writeGltf(gltf, outputPath, embed, embedImage, createDirectory, callbac
     if (createDirectory) {
         outputPath = path.join(path.dirname(outputPath), 'output', path.basename(outputPath));
     }
-    mkdirp.sync(path.dirname(outputPath));
+    fse.ensureDirSync(path.dirname(outputPath));

```

Comment:
You don't need this because `outputFile` will create a new directory if it doesn't exist.